### PR TITLE
Added support channels to contacts

### PR DESCRIPTION
**Pull request by ashwinraghav**

### CODE REVIEW

**Comment by ashwinraghav** [comment]
Failures are unrelated

File: .github/config.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: 🔥 Firebase Support
+    url: https://firebase.google.com/support/
+    about: If you have an urgent issue with your app, please contact support.
```
Comment (rosariopfernandes) [inline .github/config.yml:5]: @ashwinraghav I think you meant to create this file under `.github/ISSUE_TEMPLATE/`?
(As shown in the [Github Docs](https://docs.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository#configuring-the-template-chooser))

Comment (ashwinraghav) [reply .github/config.yml:5]: Yep. Realised that late Friday. Thanks 👍

**Pull request merged**